tests/types/test_arguments.py

```diff
@@ -1,6 +1,11 @@
 from typing import List, Optional
 
+import pytest
+
+from typing_extensions import Annotated
```

Comment:
I wonder if we should do a test just for py3.9? maybe it's not worth doing it

Reply:
Probably a good idea, will add.